codeBlock-1984/bankr
# #165438937 Refactor unit tests (TEST)

### What does this PR do?
Refactors all unit tests to ensure they are working with the integrated database and passing 

### Description of Task to be completed?
- Refactor all unit tests to work with the integrated database
- Refactor related controllers and files to ensure tests are passing

### How should this be manually tested?
1. Follow the instructions on the README file to start the development environment on your local machine.
2. In your terminal run ```npm test```. You should see all tests passing.

### What are the relevant pivotal tracker stories?
[#165438937](https://www.pivotaltracker.com/story/show/165438937)
